pkg/kubelet/kubelet_pod_lister.go

```diff
@@ -50,6 +51,8 @@ func init() {
 		port = "10250"
 	}
 	podURL = "https://" + nodeName + ":" + port + "/pods"
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	client = http.Client{}
```

Comment:
`Error: pkg/kubelet/kubelet_pod_lister.go:69:12: &http.Client{} escapes to heap`
make http client reuse